src/components/authentication/LoginPage.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import { withRouter } from 'react-router';
+import { withStyles } from '@material-ui/core/styles';
+import company from '../../images/galvanize_long.jpg';
+import galvanize from '../../images/galvanize.png';
+import microsoft from '../../images/microsoft.png';
+
+const styles = {
+  button: {
+    borderRadius: '5px',
+    color: '#ffffff',
+    display: 'flex',
+    padding: '5px 10px',
+    margin: '10px auto',
+  },
+  company: {
+    display: 'block',
+    margin: '200px auto 50px',
+    width: '300px',
+  },
+  divider: {
+    height: '35px',
+    width: '1px',
+    backgroundColor: '#ffffff',
+    margin: 'auto 10px',
+  },
+  galvanize: {
+    backgroundColor: '#280e3a',
+  },
+  logo: {
+    width: '25px',
+    margin: 'auto',
+  },
+  microsoft: {
+    backgroundColor: '#3f51b5',
+  },
+  text: {
+    margin: 'auto 50px',
+  },
+  help: {
+    textAlign: 'center',
+    fontSize: '12px'
+  },
+  highlight: {
+    color: '#483d8b',
+    marginLeft: '10px',
+  },
+  footer: {
+    color: '#ffffff',
+    backgroundColor: '#333',
+    position: 'absolute',
+    bottom: '0',
+    width: '100%',
+    padding: '10px 0',
+  },
+  license: {
+    padding: '0 25px',
+  }
+};
+
+class LoginPage extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+
+    };
+  }
+
+  render() {
+    const { classes } = this.props;
+    return <div>
+      <div>
+        <img className={classes.company} src={company} alt={'Galvanize Logo'}></img>
+        <button className={`${classes.button} ${classes.galvanize}`}>
+          <img src={galvanize} className={classes.logo} alt={'Galvanize Logo'}></img>
+          <div className={classes.divider}></div>
+          <p className={classes.text}>Sign in with Galvanize Account</p>
+        </button>
+        <button className={`${classes.button} ${classes.microsoft}`}>
+          <img src={microsoft} className={classes.logo} alt={'Microsoft Logo'}></img>
+          <div className={classes.divider}></div>
+          <p className={classes.text}>Sign in with Microsoft Account</p>
+        </button>
+        {/* <p className={classes.help}>Trouble signing in?<span className={classes.highlight}>Recover account</span></p> */}
```

Comment:
I know you used `<p>` because the buttons required background images. I'm not sure if you can use a typography in the buttons and whether that's a good way to handle it, but for normal texts let's stick with Typography :p